eslint.config.mjs

```
@@ -19,7 +19,7 @@ const compat = new FlatCompat({
 
 const eslintConfig = [
   {
-    ignores: ['**/_*/**', "**/.next/**"],
+    ignores: ['**/_*/**', , '**/dist/**', '**/node_modules/**', "**/.next/**", "**/.turbo/**"],
```

Biome (2.1.2)

[error] This array contains an empty slots..

The presences of empty slots may cause incorrect information and might be a typo.
Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)